buildSrc/src/main/kotlin/kord-module.gradle.kts

```diff
@@ -63,14 +64,14 @@ tasks {
     withType<AbstractDokkaLeafTask> {
         // see https://kotlin.github.io/dokka/<dokka version>/user_guide/gradle/usage/#configuration-options
 
-        // make sure ksp generates files before building docs
-        dependsOn(compileKotlin)
+        // include documentation generated by ksp
+        dependsOn(kspKotlin)
```

Comment:
Does this have something to do with the new Gradle version?

Reply:
Not directly, however the intention was always to only rely on the `kspKotlin` task, not the `compileKotlin` task. And because since Gradle 8.0 you are no longer allowed to have implicit task dependencies (like we had before between `sourcesJar` and `kspKotlin`) I had to have this reference to the `kspKotlin` task anyway and also used it here.